iocage/lib/ResourceUpdater.py

```diff
@@ -488,6 +496,7 @@ def _wrap_command(self, command: str) -> str:
         _command = "\n".join([
             "set +e",
             f"OUTPUT=\"$({command})\"",
+            "echo $OUTPUT",
```

Comment:
Was this part of the debugging code, or is it necessary / useful?

Reply:
The exit code of some `freebsd-update.sh` versions is `1` in case no updates were installed. This wrapper script checks whether this is the case and exits with `0` anyway, so that libiocage does not complain about the failed execution. Later on we read the output again (from iocage.lib.events.JailLaunch.stdout) and determine whether the updates were skipped or some have been newly installed.